csfunctions/events/document_release_check.py

```diff
@@ -0,0 +1,19 @@
+from typing import Literal
+
+from pydantic import BaseModel, Field
+
+from csfunctions.objects import Document, Part
+
+from .base import BaseEvent, EventNames
+from .dialog_data import DocumentReleaseDialogData
+
+
+class DocumentReleaseCheckData(BaseModel):
+    documents: list[Document] = Field(..., description="List of documents that will be released.")
+    parts: list[Part] = Field(..., description="List of parts that belong to the documents")
```

Comment:
We should maybe give it a name where it is clear that the parts are not the objects to be released. `attached_parts` ?

The same applies for the other events of documents and parts

Reply:
That would be more clear, yes.
However, for the existing events it would be a breaking change.

Reply:
At this early point of the SDK I think it is okay